openmdao/components/eq_constraints_comp.py

```diff
@@ -0,0 +1,249 @@
+"""Define the EqualityConstraintsComp class."""
+
+from __future__ import print_function, division, absolute_import
+
+from numbers import Number
+from six import iteritems
+
+import numpy as np
+
+from openmdao.core.explicitcomponent import ExplicitComponent
+
+
+class EqualityConstraintsComp(ExplicitComponent):
```

Comment:
To long of a name, plus plural isn't consistent with BalanceComp. I prefer EQConstraintComp

Reply:
done